test/collection/bimap_test.dart

```diff
@@ -40,6 +40,24 @@ void main() {
       expect(map.inverse.isNotEmpty, false);
     });
 
+    test('should support null keys', () {
+      final map = BiMap<String?, int>();
+      map[null] = 5;
+      expect(map.isEmpty, false);
+      expect(map.containsKey(null), true);
+      expect(map.containsValue(5), true);
+      expect(map.keys, contains(null));
+    });
+
+    test('should support null values', () {
+      final map = BiMap<String, int?>();
+      map[k1] = null;
+      expect(map.isEmpty, false);
+      expect(map.containsKey(k1), true);
+      expect(map.containsValue(null), true);
+      expect(map[k1], isNull);
```

Comment:
Test `map.inverse`?

Reply:
That should be pretty heavily tested already; or a sepcific map inverse test?

Reply:
My thinking was if you start with a bimap that allows nullable values, when you invert it you'll end up with one allowing nullable keys, which is a new thing. But perhaps it's already covered and I just can't tell if it is.

Reply:
Nullable keys were always allowed in Dart, though like all keys only one of them. In this case the existing documentation already covers the behaviour, it's just that we allow one additional key (or value) than we previously did -- i.e., null.